kernel/src/main/java/org/kframework/compile/AddSortInjections.java

```diff
@@ -117,6 +117,11 @@ public Boolean merge(Boolean a, Boolean b) {
         return internalAddSortInjections(body, sort);
     }
 
+    private Sort getCellSort(KLabel klabel) {
+        ConfigurationInfoFromModule configInfo = new ConfigurationInfoFromModule(mod);
```

Comment:
can we actually add this as a field to the class rather than constructing it on the fly like this? it would be much cheaper if we are calling this function many times, which we probably are.

Reply:
Fixed